src/shared/services/fields/fields.utils.ts

```diff
@@ -170,8 +192,8 @@ export class FieldsUtils {
     }
   }
 
-  private static getFixedListLabelByCodeOrEmpty(field, code) {
-    let relevantItem = code ? field.field_type.fixed_list_items.find(item => item.code === code) : '';
+  private static getFixedListLabelByCodeOrEmpty(field: CaseField, code: any): string {
```

Comment:
Just checking that there's no closer match than `any` for the `code` parameter?

Reply:
Same as above... but I'll look again. :)

Reply:
No, I take that back. I simply got the type wrong in this instance: `string` is all good.

Reply:
Done.